src/main/java/com/parasoft/demoapp/dto/GlobalPreferencesResponseDTO.java

```diff
@@ -0,0 +1,58 @@
+package com.parasoft.demoapp.dto;
+
+import com.parasoft.demoapp.model.global.preferences.*;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Set;
+
+@Data
+@NoArgsConstructor
+@Schema
+public class GlobalPreferencesResponseDTO {
+
+    private IndustryType industryType;
+
+    private WebServiceMode webServiceMode;
+
+    private String graphQLEndpoint;
+
+    private Boolean advertisingEnabled;
+
+    private Set<DemoBugEntity> demoBugs;
+
+    private Set<RestEndpointEntity> restEndPoints;
+
+    private Boolean useParasoftJDBCProxy;
+
+    private String parasoftVirtualizeServerUrl;
+
+    private String parasoftVirtualizeServerPath;
+
+    private String parasoftVirtualizeGroupId;
+
+    private MqType mqType;
+
+    private MqConfigDTO activeMqConfig = new MqConfigDTO();
+
+    private MqConfigDTO kafkaConfig = new MqConfigDTO();
+
+    public GlobalPreferencesResponseDTO(GlobalPreferencesEntity globalPreferencesEntity) {
+        this.setIndustryType(globalPreferencesEntity.getIndustryType());
+        this.setWebServiceMode(globalPreferencesEntity.getWebServiceMode());
+        this.setGraphQLEndpoint(globalPreferencesEntity.getGraphQLEndpoint());
+        this.setAdvertisingEnabled(globalPreferencesEntity.getAdvertisingEnabled());
+        this.setDemoBugs(globalPreferencesEntity.getDemoBugs());
+        this.setRestEndPoints(globalPreferencesEntity.getRestEndPoints());
+        this.setUseParasoftJDBCProxy(globalPreferencesEntity.getUseParasoftJDBCProxy());
+        this.setParasoftVirtualizeServerUrl(globalPreferencesEntity.getParasoftVirtualizeServerUrl());
+        this.setParasoftVirtualizeServerPath(globalPreferencesEntity.getParasoftVirtualizeServerPath());
+        this.setParasoftVirtualizeGroupId(globalPreferencesEntity.getParasoftVirtualizeGroupId());
+        this.setMqType(globalPreferencesEntity.getMqType());
+        this.getActiveMqConfig().setOrderServiceListenOn(globalPreferencesEntity.getOrderServiceDestinationQueue());
```

Comment:
I think these two values need to be swapped at line 53 and 54.
![image](https://user-images.githubusercontent.com/35082985/205395149-afad2e31-c635-4b1c-994a-42863c6cf6fb.png)